test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -739,6 +743,54 @@ private HapiApiSpec noStakePeriodStartIfNotStakingToNode() {
                         getContractInfo(contract).has(contractWith().noStakePeriodStart()));
     }
 
+    private HapiApiSpec hollowAccountCreationWithCryptoTransfer() {
+        final var hollowAccountCreateSponsor = "hollowAccountCreateSponsor";
+        return defaultHapiSpec("HollowAccountCreationWithCryptoTransfer")
+                .given(
+                        UtilVerbs.overriding("lazyCreation.enabled", "true"),
+                        newKeyNamed(SECP_256K1_SOURCE_KEY).shape(SECP_256K1_SHAPE),
+                        cryptoCreate(hollowAccountCreateSponsor)
```

Comment:
Is the sponsor account hollow? It has assigned key, so probably it's a regular account? If so, we can change the name of the variable to be less misleading. I believe a hollow account would be created for the `evmAddress` on the `to` side of the `cryptoTransfer`?

Reply:
I see. I believe this would be more suitable name, indeed.